src/api/teams.rs

```diff
@@ -12,6 +12,10 @@ use crate::{
 
 #[post("/teams", data = "<team>")]
 pub async fn create(user: User, team: Json<NewTeam>, conn: DbConn) -> Result<Json<Team>, Status> {
+    if !validate_slug(&team.slug) {
+        return Err(Status::UnprocessableEntity);
```

Comment:
this should convert the slug to an acceptable version

Reply:
or rather, add an option?

Reply:
that creates an unexpected side effect with API usage; if your parameters are invalid, the server should tell you so